components/org.wso2.carbon.identity.oauth.uma.resource.endpoint/src/test/java/org/wso2/carbon/identity/oauth/uma/resource/endpoint/impl/ResourceRegistrationApiServiceImplExceptionTest.java

```diff
@@ -49,6 +55,12 @@
     @Mock
     private MessageContext mockMessageContext;
 
+    @Mock
+    private AuthenticationContext mockAuthenticationContext;
```

Comment:
Avoid mocking unnecessarily. You can set almost all the things required to authentication context. Hence we should be able to get things done without mocking this.